.travis.yml

```diff
@@ -13,55 +31,35 @@ before_install:
   - openssl aes-256-cbc -K $encrypted_d52b986e6313_key -iv $encrypted_d52b986e6313_iv -in $TRAVIS_BUILD_DIR/test/config/mysql.config.enc -out $TRAVIS_BUILD_DIR/test/config/mysql.config -d
   - openssl aes-256-cbc -K $encrypted_d52b986e6313_key -iv $encrypted_d52b986e6313_iv -in $TRAVIS_BUILD_DIR/test/config/sqlserver.config.enc -out $TRAVIS_BUILD_DIR/test/config/sqlserver.config -d
 
-  # Add custom PPAs from cartodb
-  - sudo add-apt-repository -y ppa:cartodb/postgresql-9.5
-  - sudo add-apt-repository -y ppa:cartodb/odbc
-  - sudo apt-get update
-
-  # Needed for the compilation of odbc_fdw.c
-  - sudo apt-get -y install unixodbc-dev
-
-  # Force instalation of libgeos-3.5.0 (presumably needed because of existing version of postgis)
-  #- sudo apt-get -y install libgeos-3.5.0=3.5.0-1cdb2
-
   # Install postgres db and build deps
   - sudo /etc/init.d/postgresql stop # stop travis default instance
-  - sudo apt-get -y remove --purge postgresql-9.1
   - sudo apt-get -y remove --purge postgresql-9.2
   - sudo apt-get -y remove --purge postgresql-9.3
   - sudo apt-get -y remove --purge postgresql-9.4
   - sudo apt-get -y remove --purge postgresql-9.5
+  - sudo apt-get -y remove --purge postgresql-9.6
   - sudo rm -rf /var/lib/postgresql/
   - sudo rm -rf /var/log/postgresql/
   - sudo rm -rf /etc/postgresql/
   - sudo apt-get -y remove --purge postgis-2.2
   - sudo apt-get -y autoremove
 
-  - sudo apt-get -y install postgresql-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install postgresql-server-dev-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install odbcinst
-  - sudo apt-get -y install odbc-postgresql
+  - sudo apt-get -y install postgresql-9.5
```

Comment:
if we're using the official package then maybe there's no point in uninstalling and then re-installing it

Reply:
yep, you're right. Clearing.